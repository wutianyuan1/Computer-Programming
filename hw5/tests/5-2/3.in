4 3 1
Umi 20 aa 43
Honoka 43253 65789 87912
Kotori 1.7 foo 44