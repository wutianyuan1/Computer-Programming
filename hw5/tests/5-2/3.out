/--------------------------------\
| Kotori | 1.7   | foo   | 44    |
|--------|-------|-------|-------|
| Umi    | 20    | aa    | 43    |
|--------|-------|-------|-------|
| Honoka | 43253 | 65789 | 87912 |
\--------------------------------/
